src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -404,7 +404,9 @@ private function createQueryBuilder(string $alias = 'm'): QueryBuilder
 
         $alias .= '.';
 
-        if (!$this->driverConnection->getDatabasePlatform() instanceof OraclePlatform) {
+        // FirebirdPlatform ist not part of Doctrine/Dbal Package
```

Comment:
which package is it part of then?
why can't we check for the FQCN?

Reply:
It's part of https://github.com/satwareAG/doctrine-firebird-driver. 
We can check for \Satag\DoctrineFirebirdDriver\Platforms\FirebirdPlatform